modules/sum-transformers/clients/sum_meta.go

```diff
@@ -0,0 +1,45 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2022 SeMI Technologies B.V. All rights reserved.
+//
+//  CONTACT: hello@semi.technology
+//
+
+package clients
+
+import (
+	"context"
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/pkg/errors"
+)
+
+func (s *sum) MetaInfo() (map[string]interface{}, error) {
+	req, err := http.NewRequestWithContext(context.Background(), "GET", s.url("/meta"), nil)
+	if err != nil {
+		return nil, errors.Wrap(err, "create GET meta request")
```

Comment:
I'd not use capital letters in the error message since it is not considered a best practice as far as I know